### PR TITLE
JRNYS-167: Handling of Center Overlay template

### PR DESCRIPTION
**Pull request by wwest-branch**
Implemented changes to handle center_overlay template type. Added type: center_overlay to metadata in html file for template located in Dashboard repo. I believe this implementation should be safe given that changes are only happening to center_overlay type templates.

### CODE REVIEW

File: dist/build.js [hunk 1]
```diff
@@ -958,7 +958,7 @@ goog.json.Serializer.prototype.serializeObject_ = function(a, b) {
   b.push("}");
 };
 // Input 2
-var config = {app_service_endpoint:"https://app.link", link_service_endpoint:"https://bnc.lt", api_endpoint:"https://api2.branch.io", version:"2.59.0"};
+var config = {app_service_endpoint:"https://app.stage.branch.io", link_service_endpoint:"https://link.stage.branch.iot", api_endpoint:"https://nightlyregression.api.beta.branch.io", version:"2.59.0"};
```
Comment (jwangler-branch) [inline dist/build.js:961]: You have to recompile with the prod system URL

File: src/journeys_utils.js [hunk 2]
```diff
@@ -360,8 +363,14 @@ function generateIframeOuterCSS() {
 						'transition: all 0' + (journeys_utils.animationSpeed / 1000) + 's ease;';
 	}
 
+	var isCenterOverlay = metadata.type === "center_overlay";
+
+	if (isCenterOverlay) {
+		document.body.style.background = "rgba(0,0,0,0.4)";
```
Comment (jwangler-branch) [inline src/journeys_utils.js:369]: Could be in the future, that the customer whats to change the overlay color, from shade gray to something special. Shade gree or like that. But for now it's ok to do it in this way.

File: src/journeys_utils.js [hunk 3]
```diff
@@ -460,6 +469,13 @@ journeys_utils.animateBannerEntrance = function(banner, cssIframeContainer) {
 				// check if safeAreaRequired is true or not
 				if (journeys_utils.journeyLinkData && journeys_utils.journeyLinkData['journey_link_data'] && !journeys_utils.journeyLinkData['journey_link_data']['safeAreaRequired']) {
 					banner.style.bottom = '0';
+
+					if (metadata.type === "center_overlay") {
+						banner.style.bottom = "140px";
+                        banner.style.width = "94%";
+                        banner.style.borderRadius = "20px";
```
Comment (jwangler-branch) [inline src/journeys_utils.js:476]: We should move this to separate function like the **else** case 

````
else {
					journeys_utils._dynamicallyRepositionBanner();
				}
````
lets name is like journeys_utils._centerOverlay();

File: src/journeys_utils.js [hunk 4]
```diff
@@ -460,6 +469,13 @@ journeys_utils.animateBannerEntrance = function(banner, cssIframeContainer) {
 				// check if safeAreaRequired is true or not
 				if (journeys_utils.journeyLinkData && journeys_utils.journeyLinkData['journey_link_data'] && !journeys_utils.journeyLinkData['journey_link_data']['safeAreaRequired']) {
 					banner.style.bottom = '0';
+
+					if (metadata.type === "center_overlay") {
```
Comment (jwangler-branch) [inline src/journeys_utils.js:473]: Lets move "center_overlay" to const if possible

File: dist/build.js (hunk 1 above)
Comment (wwest-branch) [reply dist/build.js]: good catch, done

File: src/journeys_utils.js (hunk 3 above)
Comment (wwest-branch) [reply src/journeys_utils.js]: nice, good suggestion, done

File: src/journeys_utils.js (hunk 4 above)
Comment (wwest-branch) [reply src/journeys_utils.js]: done

**Comment by jwangler-branch** [comment]
LGM 👍

**Review by jwangler-branch — APPROVED** [review]

**Pull request merged**